portfolio/src/main/webapp/index.html

```diff
@@ -3,17 +3,55 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <!-- <meta name="viewport" content="width=device-width, initial-scale=1.0"> -->
     <link rel="stylesheet" href="style.css">
+    <!-- <script src="https://kit.fontawesome.com/a076d05399.js" crossorigin="anonymous"></script> -->
     <script src="script.js"></script>
   </head>
   <body>
+    <nav>
+        <input type="checkbox" id="check">
+        <label for="check" class="checkbtn">
+            <i class="fas fa-bars"></i>
+        </label>
+        <label class="logo">Brandocean ⚡️</label>
+        <ul>
+            <li><a class="active" href="#">Home</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Projects</a></li>
+            <li><a href="#">Experience</a></li>
+            <li><a href="#">Contact</a></li>
+            <li><a href="#">Feedback</a></li>
+            <li><a href="Experiments.html">Experiments</a></li>
+        </ul>
+    </nav>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Brandon's Portfolio</h1>
+      <img class="center" src="/images/Brandocean.jpg" width="300" height="300"/>
+      <h2>Who am I?</h2>
+      <p>Hi! I'm Brandon, I'm from Morelos, Mexico. 
+          I joined to SPS program to improve my skills, meet new people and have fun because I love my bachelor degree. 
+          I like watching anime (my favourite Danmachi) and have fun with my friends.</p>
+      <h2>What makes me me?</h2>
+      <p>I am a noble person, persistent, friendly, methodical, enterprising and with leadership skills. 
+          Always having in my mind the phrase “nothing is impossible, just a little challenging”</p>
+      <h2>My projects</h2>
+      <ul>
+        <li><b>Plant monitoring with IoT:</b> October-November 2021. Web Service
```

Comment:
[3.1.5](https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns): Separate structure from presentation from behavior.

Here and elsewhere.

Instead of `<b>` you could use [`<strong>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong):
```suggestion
        <li><strong>Plant monitoring with IoT:</strong> October-November 2021. Web Service
```
The difference lies in the purpose as [`<b>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b) is mostly a styling element.